composer.json

```diff
@@ -10,6 +10,7 @@
         "filament/filament": "^3.0",
         "guzzlehttp/guzzle": "^7.8",
         "laravel/framework": "^10.30",
+        "laravel/pail": "^1.0",
```

Comment:
This is beyond the scope of the issue https://github.com/gomzyakov/laravel-blog/issues/146. Please take it away.